### PR TITLE
Support building python bundles

### PR DESCRIPTION
**Pull request by leeweisberger**
## Description
Support building python bundles using a universal shim.

## Test plan
Unit tests to ensure that we didn't break existing functionality and that new bundle based functionality is supported.

### CODE REVIEW

File: pkg/build/universal-node-shim.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ async function main() {
   const entrypointFunc = process.argv[3];
   const params = process.argv[4];
 
-  // TODO: needs environment variables to be set properly
```
Comment (ameyakhare) [inline pkg/build/universal-node-shim.js:18]: oops

File: pkg/build/python_test.go [hunk 2]
```diff
@@ -69,3 +69,152 @@ func TestPythonBuilder(t *testing.T) {
...
+			ParamValues: map[string]interface{}{
+				"hello": "world",
+			},
+			BuildContext: BuildContext{
+				Type:    PythonBuildType,
+				Version: BuildTypeVersionPython37,
+			},
+			BundleRuns: []BundleTestRun{
+				{
+					RelEntrypoint: "main.py",
```
Comment (ameyakhare) [inline pkg/build/python_test.go:93]: why is there no export name for noninline tasks?

**Review by ameyakhare — APPROVED** [review]
looks fine

File: pkg/build/python_test.go (hunk 2 above)
Comment (leeweisberger) [reply pkg/build/python_test.go:93]: This still works (both python and node) if entrypointFunc is not defined

**Pull request merged**